src/components/Filters.js

```diff
@@ -1,39 +1,56 @@
-import { useEffect } from "react";
-import { BsChevronDown } from "react-icons/bs";
 import { MdOutlineKeyboardAlt } from "react-icons/md";
 
 
-const Filters = ({ tag, handleTag, data, tagsArray, filter }) => {
-  useEffect(() => {
-    if (tag !== "") {
-    }
-
-  }, [tag])
+const Filters = ({ tag, handleTag, resetData, tagsArray, filter, handleMidTag, midTag, handleFilterInput, filterInput }) => {
+  // console.log(tag)
+  
   const handleCategoryChange = (e) => {
+    resetData()
+    handleMidTag("")
     handleTag(e.target.value)
+    handleFilterInput("")
```

Comment:
Make a function called resetFilters and reset individual filter in the function:

const resetFilters = {
    handleMidTag("")
    handleTag("")
    handleFilterInput("")
}

By doing this you don't have to repeat yourself

Reply:
I have to use all three only once. Should I still put them inside a function and call it once?